src/python/test/test_dxpy.py

```diff
@@ -2592,6 +2592,42 @@ def test_get_handler(self):
         self.assertEqual(handler._name, "swiss-army-knife")
         self.assertEqual(handler._alias, "1.0.0")
 
+    def test_describe_data_objects(self):
+        objects = []
+        types = []
+        tags = []
+        objects.append(dxpy.new_dxrecord())
+        types.append('record')
+        tags.append([])
+        objects.append(dxpy.new_dxfile())
+        types.append('file')
+        tags.append([])
+        objects.append(dxpy.new_dxworkflow())
+        types.append('workflow')
+        tags.append(['my_tag'])
+        objects[-1].add_tags(tags[-1])
+        
+        # Should be able to handle a mix of raw ids and dxlinks.
+        ids = [o.get_id() for o in objects]
+        desc = dxpy.describe(ids)
+
+        self.assertEqual(len(ids), len(desc))
+        for i in xrange(len(desc)):
+            self.assertEqual(desc[i]["project"], self.proj_id)
+            self.assertEqual(desc[i]["id"], ids[i])
+            self.assertEqual(desc[i]["class"], types[i])
+            self.assertEqual(desc[i]["types"], []) 
+            self.assertTrue("created" in desc[i]) 
```

Comment:
Do you mean change
self.assertEqual(desc[i]["id"], ids[i])
to 
self.assertIn(desc[i]["id"], ids)
?

That would no longer test that the order is proper though, right?  We want to make sure that the order of the descriptions we get back matches the order of the id's we send in, right?